api/python_api/kubeflow_trainer_api/models/trainer_v1alpha1_model_initializer.py

```
@@ -26,11 +26,11 @@
 
 class TrainerV1alpha1ModelInitializer(BaseModel):
     """
-    DatasetInitializer represents the desired configuration to initialize pre-trained model. The DatasetInitializer spec will override the runtime Job template which contains this label: `trainer.kubeflow.org/trainjob-ancestor-step: dataset-initializer`
+    ModelInitializer represents the desired configuration to initialize pre-trained model. The ModelInitializer spec will override the runtime Job template which contains this label: `trainer.kubeflow.org/trainjob-ancestor-step: dataset-initializer`
     """ # noqa: E501
```

Ruff (0.14.4)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)